### ISSUE TITLE
Support maximizing objective in `workon`, `report_objective`, etc.

### ISSUE THREAD

**Issue by lebrice** [Epistimio/orion#873]
Instead of asking users to do something, we should do it ourselves.
This is the kind of super basic thing that should absolutely be supported natively by `Orion`, without having to resort to a cheap `*-1` trick that would make all commands report non-sensical negative accuracy values, etc.

**Comment by bouthilx** [comment]
Yeah our position was that we were making an MVP and that was quite easy for the users to handle. I think Oríon is clearly passed the MVP stage now, so it's about time. :sweat_smile: It would be great to handle this the same way we do with spaces, so that we invert objectives to be all minimized when passed to the algorithms. That would make algorithm implementation simpler.
